### PR TITLE
Update casestudy.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
Note, for step #6 in the case study process, I thought it would be easier to add a link to a sample banner after the first case study blog post banner has been created.

File: source/process/casestudy.rst [hunk 1]
```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval.
+5. Mattermost case study coordinator politely checks in with the customer every 2 business days for an estimate of the draft case study review time.
+6. The case study coordinator also commissions a banner with a clear heading and a nice background image to be included in the case study blog post.
```
Comment (jasonblais) [inline source/process/casestudy.rst]: Can we make the following more obvious to a new customer on what is expected?
 - "clear" heading --> would there be an example of one?
 - "nice" background image --> guidelines on what the image should have? Guidelines on sizing, etc?

I think a link to an example banner is fine to add as well

File: source/process/casestudy.rst [hunk 2]
```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
...
+7. The case study coordinator creates a draft blog post (unpublished) on the Mattermost staging server. The draft will then be reviewed for formatting, grammar, and so on.
```
Comment (jasonblais) [inline source/process/casestudy.rst]: Does this step happen after we get the post from the customer?

Other minor notes:
1. Is `(unpublished)` needed?
2. Is `Mattermost staging server` needed? (if this page is for customers, probably not)
3. Can we link to style guidelines? E.g. what formatting should it follow? Perhaps the guidelines for blog posts?

File: source/process/casestudy.rst [hunk 3]
```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
...
+8. When the customer has completed a review of the draft case study, the draft blog post can be updated to be published.
```
Comment (jasonblais) [inline source/process/casestudy.rst]: I feel the current flow isn't quite clear:
 1. draft is sent to customer
 2. in the meantime banner is commissioned (this is okay)
 3. in the meantime post is drafted and reviewed for grammar
 4. then customer sends it back, and the draft is updated (and reviewed again for grammar?)

Maybe 3 and 4 above should we swapped?

Comment (jasonblais) [inline source/process/casestudy.rst]: `can be updated to be published.` Maybe this is part of step 9?

File: source/process/casestudy.rst [hunk 4]
```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
...
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval.
+5. Mattermost case study coordinator politely checks in with the customer every 2 business days for an estimate of the draft case study review time.
+6. The case study coordinator also commissions a banner with a clear heading and a nice background image to be included in the case study blog post.
+7. The case study coordinator creates a draft blog post (unpublished) on the Mattermost staging server. The draft will then be reviewed for formatting, grammar, and so on.
+8. When the customer has completed a review of the draft case study, the draft blog post can be updated to be published. 
+9. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
```
Comment (jasonblais) [inline source/process/casestudy.rst]: I guess step 8 has the customer review/approval?

And step 9 is then just publishing it with the banner?

File: source/process/casestudy.rst (hunk 1 above)
Comment (amyblais) [inline source/process/casestudy.rst]: Jason, sorry I made a note above that I will add a link to a sample when the first one is done.

Comment (jasonblais) [inline source/process/casestudy.rst]: I agree the sample is good, but we might still want to clarify the meanings of "clear" and "nice"

Example: Should the image be a screenshot/photo of the company headquarters?

File: source/process/casestudy.rst (hunk 3 above)
Comment (amyblais) [inline source/process/casestudy.rst]: The draft (in a Google doc) sent to the customer is differrent from the blog post draft (in WordPress) - I tried to tweak these steps but the ordering is fine

File: source/process/casestudy.rst [hunk 5]
```diff
@@ -14,8 +14,15 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval (in a shared Google document).
+5. Mattermost case study coordinator:
+
+   -  politely follows up with the customer every 2 business days for an estimate of the draft case study review time
```
Comment (it33) [inline source/process/casestudy.rst]: Propose capitalizing after bullet points. I don't know if this is correct, but it's what we do in other docs.

File: source/process/casestudy.rst [hunk 6]
```diff
@@ -14,8 +14,15 @@ The process is approximately as follows:
...
+   -  commissions a banner to be featured in the case study blog post
+   -  creates a draft blog post in WordPress. The draft is then reviewed for formatting, grammar, and other `style guidelines <https://docs.mattermost.com/process/marketing-guidelines.html>`_
+
+6. When the customer has completed a review of the draft case study and approved it, the draft blog post is updated.
```
Comment (it33) [inline source/process/casestudy.rst]: We should have a step where customer feedback is incorporated into draft blog post in Wordpress and a final draft (using preview mode in Wordpress) is sent to customer as FYI prior to publishing.

File: source/process/casestudy.rst (hunk 5 above)
Comment (it33) [inline source/process/casestudy.rst]: Also, regarding @jasonblais question on why 2 days, propose we update so it's either checking in the day before the customer says they'll review, to confirm review is coming, or every 2 days if we don't have an estimated review date, just to keep up the energy. 

@amyblais I'd propose softening the language so 2 days is more a suggestion than a rule and coordinator should use their judgement.

**Pull request merged**